Sources/SwiftParser/Expressions.swift

```diff
@@ -2487,7 +2487,7 @@ extension Parser.Lookahead {
 extension Parser.Lookahead {
   // Helper function to see if we can parse member reference like suffixes
   // inside '#if'.
-  fileprivate mutating func isAtStartOfPostfixExprSuffix() -> Bool {
+  fileprivate mutating func atStartOfPostfixExprSuffix() -> Bool {
```

Comment:
I would keep the functions in `Lexer.Cursor` in the `isAt` style. The main checking function of `Lexer.Cursor` also is `is(offset: Int = 0, at: CharcterByte)`, so that fits with the `isAt` style. The parser just uses `at(_: TokenSpec)`.

Reply:
Got it! One sec, I'll clean this up.